src/mmframe.cpp

```diff
@@ -1782,6 +1782,12 @@ void mmGUIFrame::createMenu()
     menuItemReportBug->SetBitmap(mmBitmap(png::BUG));
     menuHelp->Append(menuItemReportBug);
 
+    wxMenuItem* menuItemCrowdin = new wxMenuItem(menuHelp, MENU_CROWDIN
+        , _("Improve translation")
+        , _("Add/Improve GUI Translation"), wxITEM_MAX);
```

Comment:
```suggestion
        , _("Improve GUI Translation"), wxITEM_MAX);
```
Is there any way to add new translation by a user?